e2e/e2e_test.go

```diff
@@ -33,9 +36,13 @@ func initClient(t *testing.T) *api.ClientWithResponses {
 }
 
 func TestMain(m *testing.M) {
-	// Make sure OpenMeter is ready
-	// TODO: replace this with some sort of health check
-	time.Sleep(15 * time.Second)
+	wait := os.Getenv("TEST_WAIT_ON_START")
+
+	if b, err := strconv.ParseBool(wait); err == nil && b {
```

Comment:
you can workaround this to put a `testutils.EventuallyWithTf` before that calls healthcheck or queries meter